include/screencast_common.h

```diff
@@ -83,6 +94,7 @@ struct xdpw_screencast_instance {
 	uint32_t refcount;
 	struct xdpw_screencast_context *ctx;
 	bool initialized;
+	struct xdpw_frame simple_frame;
```

Comment:
Why is this named "simple"?

Reply:
Because the old one was. Renamed it to current_frame, another good pick might be working_frame. This struct is the central part containing all information, which has to be transferred between wlroots and pipewire.